finality-provider/service/fp_instance.go

```diff
@@ -724,12 +733,60 @@ func (fp *FinalityProviderInstance) CommitPubRand(targetBlockHeight uint64) (*ty
 
 	// Update metrics
 	fp.metrics.RecordFpRandomnessTime(fp.GetBtcPkHex())
-	fp.metrics.RecordFpLastCommittedRandomnessHeight(fp.GetBtcPkHex(), lastCommittedHeight)
```

Comment:
we'd better also record the local last committed height to the metrics

Reply:
yes it's still recorded. just moved due to refactoring